### ISSUE TITLE
Suggestions [Holography for Active Sensing]

### ISSUE THREAD

**Issue by btilmon** [bchao1/awesome-holography#2]
# Topics in Holography Display Systems 

* [Energy-Efficient Adaptive 3D Sensing](https://btilmon.github.io/e3d.html) (CVPR 2023) uses a spatial light modulator for energy-efficient active depth sensing in computer vision. I think this could warrant a new subsection along the lines of "Holography for Active Sensing" or similar. The code base [holoCu](https://github.com/btilmon/holoCu) for "Energy-Efficient Adaptive 3D Sensing" provides a CUDA implementation and simulator of the Fresnel Holography technique used in the paper. The code could be valuable as a research resource since it simulates various 3D sensors on a spatial light modulator, and as an implementation resource since it has OpenGL-CUDA interop implemented to compute and render phase maps to the spatial light modulator in real time. It also runs on limited computing platforms like NVIDIA Jetson Nano. 

Related work here is [Holocurtains: Programming Light Curtains via Binary Holography](https://imaging.cs.cmu.edu/holocurtains/) (CVPR 2022).

**Comment by bchao1** [comment]
Yes, I am aware of both works. This is indeed a very prominent application of holography. 

I will add a subsection but hope to include more papers. Do you know any other papers on holography for structured illumination, 3D scanning, or anything along those lines? 

Thanks.

**Comment by btilmon** [comment]
Sounds great! Related work is sparse, it appears most works focus on structured light for microscopy, not "in the wild" active 3D sensing, but I managed to find the following work:
*  [Light field microscopy based on structured light illumination](https://opg.optica.org/ol/viewmedia.cfm?uri=ol-46-14-3424&seq=0) (Optics Letters 2021)
* [Holography-based structured light illumination for temporal focusing microscopy](https://opg.optica.org/ol/viewmedia.cfm?uri=ol-46-13-3143&seq=0) (Optics Letters 2021)
